internal/common/backoff/retry.go

```diff
@@ -103,16 +106,40 @@ Retry_Loop:
 			return err
 		}
 
-		// Check if the error is retryable
-		if isRetryable != nil && !isRetryable(err) {
```

Comment:
`isRetryable == nil` was only true in tests, so what was changed and now it's just assumed to exist in all cases.